README.md

```diff
@@ -1,12 +1,31 @@
-# openSUSE and Tini image
+# openSUSE Base image
 
-This project builds an openSUSE-based image which includes Tini. It can be used as a base image by projects which require Tini.
+This project builds an openSUSE-based image which includes Tini, cURL and the PostgreSQL client.
 
 ### Tini
 [Tini](https://github.com/krallin/tini) is pre-installed in the container.  If the image entrypoint is not overwritten then it will be automatically used.
 
+### PostgreSQL Client
+[PostgreSQL](https://wiki.postgresql.org/wiki/Main_Page) Client is pre-installed in the container. This enables projects to automatically install required PostgreSQL databases if required.
+
 ### Startup Scripts
 Any executable scripts added to the `/startup/startup.d/` directory will be automatically run each time the container is started (assuming the image entrypoint is not overwritten).
 
+#### Database Check and Create Script
```

Comment:
It doesn't make sense to have this as a subheading under 'Startup Scripts' but for the Certificate Installation to be a level up.

Perhaps we should have a new 'Pre-Installed Startup Scripts' section and describe both scripts under it.